sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -790,4 +791,13 @@ class APISpec extends StorageSpec {
             }
         }
     }
+
+    static Mono<String> getAuthToken() {
+        if (env.testMode == TestMode.PLAYBACK) {
+            return Mono.just("recordingBearerToken")
```

Comment:
1. How does this work? Does playback check if headers are matching or does it only care about URI?
2. We should make sure that source auth header is redacted in recordings. Could you check that?

Reply:
Playback doesn't care about oauth tokens. we just need to have some sort of string for the SDK to work and we don't want to actually try and fetch a token in playback mode.

I see no bearer tokens. I'm not sure what in our infra sets this but the outcomes appear to be fine.